docker-compose.yaml

```diff
@@ -0,0 +1,41 @@
+version: "3"
+
+services:
+  todo_app:
+    build:
+      context: .
+      target: development
+    ports:
+      - 5000:80
+    volumes:
+      - type: bind
+        source: todo_app
+        target: /app/todo_app/todo_app
+    env_file:
+      - .env
+
+  todo_app_unit_test_runner:
+    build:
+      context: .
+      target: test
+    volumes:
+      - type: bind
+        source: todo_app
+        target: /app/todo_app/todo_app
```

Comment:
There's definitely an argument it's better to write out explicitly like you have, but worth comparing with the shorthand syntax:
```yml
      - ./todo_app:/app/todo_app/todo_app
```

Just by virtue of being a path rather than a name it can derive the volume type